pkg/cnab/config_adapter/testdata/porter-with-parameters.yaml

```diff
@@ -25,15 +23,12 @@ parameters:
       - red
       - purple
       - pink
-    required: true
```

Comment:
I don't understand why we can't make a parameter required anymore. Can you walk me through that?

If it's because of a technical decision (and not because of a limitation of the spec), let's brainstorm how to get that functionality back.

Reply:
I dropped this due to a conflict of the embedded [Schema struct](https://github.com/deislabs/cnab-go/blob/master/bundle/definition/schema.go#L55) from deislabs/cnab-go.  I though it might be a candidate to drop as I didn't see its usefulness: if a parameter has no default, it would need to be required (and still is, since we do set it so in the resulting `bundle.json` in this case), but if a parameter does have a default, I didn't understand the logic of being able to deem it as required (it made better sense to be optional/non-required).

Nevertheless, opinions aside, let's brainstorm on keeping this functionality assuming we want it to stay put.  Since, as mentioned above, a `required` field is already being used when unmarshaling (by the embedded Schema definition), we could rename it to `isRequired` for an easy re-addition?

Reply:
Ah okay, no I wasn't trying to fight to keep it in. 😁 I just wasn't sure if we needed to keep it or not. Your explanation makes sense for why you removed it (default does all the work for us).